Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -0,0 +1,191 @@
+---
+title: CephCluster Network Providers
+---
+
+!!! Note
+    This is an advanced networking topic.
+    [General CephCluster networking is linked here.](./ceph-cluster-crd.md#network-configuration-settings)
+
+Rook deploys CephClusters using Kubernetes' software-defined networks by default. This is simple for
+users, provides necessary connectivity, and has good node-level security. However, this comes at the
+expense of additional latency, and the storage network must contend with Kubernetes applications for
+network bandwidth. It also means that Kubernetes applications coexist on the same network as Ceph
+daemons and can reach the Ceph cluster easily via network scanning.
+
+Rook allows selecting alternative network providers to address some of these downsides, sometimes at
+the expense of others. Selecting alternative network providers is an advanced topic.
+
+## Ceph Networking Fundamentals
+
+Ceph daemons can operate on up to two distinct networks: public, and cluster.
+
+Ceph daemons always use the public network. The public network is used for client communications
+with the Ceph cluster (reads/writes). Rook configures this as the Kubernetes pod network by default.
+Ceph-CSI uses this network for PVCs.
+
+The cluster network is optional and is used to isolate internal Ceph replication traffic. This
+includes additional copies of data replicated between OSDs during client reads/writes. This also
+includes OSD data recovery (re-replication) when OSDs or nodes go offline. If the cluster network is
+unspecified, the public network is used for this traffic instead.
+
+Refer to [Ceph networking documentation](https://docs.ceph.com/docs/master/rados/configuration/network-config-ref/)
+for deeper explanations of any topics.
+
+## Specifying Ceph Network Selections
+`network.addressRanges`
+
+This configuration is always optional but is important to understand.
```

Comment:
Added text to clarify a misconception about this set of options that at least 2 users have had.